doc/libf3d/OPTIONS.md

```diff
@@ -55,346 +60,402 @@ CLI: `--up`.
+
 CLI: `--texture-matcap`.
 
 ### `model.color.opacity` (_double_, optional)
 
 Set _opacity_ on the geometry. Usually used with Depth Peeling option. Multiplied with the `model.color.texture` when present. Model specified by default.
+
 CLI: `--opacity`.
 
 ### `model.color.rgb` (_color_, optional)
 
 Set a _color_ on the geometry. Multiplied with the `model.color.texture` when present. Model specified by default.
+
 CLI: `--color`.
 
 ### `model.color.texture` (_path_, optional)
 
 Path to a texture file that sets the color of the object. Will be multiplied with rgb and opacity. Model specified by default.
+
 CLI: `--texture-base-color`.
 
 ### `model.emissive.factor` (_color_, optional)
 
 Multiply the emissive color when an emissive texture is present. Model specified by default.
+
 CLI: `--emissive-factor`.
 
 ### `model.emissive.texture` (_path_, default: ``)
 
 Path to a texture file that sets the emitted light of the object. Multiplied with the `model.emissive.factor`. Model specified by default.
+
 CLI: `--texture-emissive`.
 
 ### `model.material.metallic` (_double_, optional)
 
 Set the _metallic coefficient_ on the geometry (0.0-1.0). Multiplied with the `model.material.texture` when present. Model specified by default.
+
 CLI: `--metallic`.
 
 ### `model.material.roughness` (_double_, optional)
 
 Set the _roughness coefficient_ on the geometry (0.0-1.0). Multiplied with the `model.material.texture` when present. Model specified by default.
+
 CLI: `--roughness`.
 
 ### `model.material.base_ior` (_double_, optional)
 
 Set the _index of refraction of the base layer_ (1.0-2.5). Model specified by default.
+
 CLI: `--base-ior`.
 
 ### `model.material.texture` (_path_, optional)
 
 Path to a texture file that sets the Occlusion, Roughness and Metallic values of the object. Multiplied with the `model.material.roughness` and `model.material.metallic`, set both of them to 1.0 to get a true result. Model specified by default.
+
 CLI: `--texture-material`.
 
 ### `model.normal.scale` (_double_, optional)
 
 Normal scale affects the strength of the normal deviation from the normal texture. Model specified by default.
+
 CLI: `--normal-scale`.
 
 ### `model.normal.texture` (_path_, optional)
 
 Path to a texture file that sets the normal map of the object. Model specified by default.
+
 CLI: `--texture-normal`.
 
 ### `model.scivis.enable` (_bool_, default: `false`)
 
 _Color by an array_ present in on the data. If `model.scivis.array_name` is not set, the first available array will be used.
+
 CLI: `--scalar-coloring`.
 
 ### `model.scivis.cells` (_bool_, default: `false`)
 
 Color the data with value found _on the cells_ instead of points
+
 CLI: `--cells`.
 
 ### `model.scivis.discretization` (_int_, optional)
 
 Set how many distinct colors will be used from the colormap. [1, N] will discretize, values outside that range will result in smooth shading.
```

Comment:
`[1, N]` is confusing but not sure how to phrase it. Does it mean strictly positive values as in `x ≥ 1`?